docs/reference/troubleshoot.md

```diff
@@ -80,13 +80,43 @@ namespace. To get the current Crossplane logs, run the following:
 kubectl -n crossplane-system logs -lapp=crossplane
 ```
 
-Remember that much of Crossplane's functionality is provided by providers. You
-can use `kubectl logs` to view provider logs too.
-
 > Note that Crossplane emits few logs by default - events are typically the best
 > place to look for information about what Crossplane is doing. You may need to
-> restart Crossplane (or your provider) with the `--debug` flag if you can't 
-> find what you're looking for.
+> restart Crossplane with the `--debug` flag if you can't find what you're
+> looking for.
+
+## Provider Logs
```

Comment:
how about adding this new section Provider Logs (and the other one Pausing Providers) to the TOC links at the top of the page?